src/Compilers/CSharp/Portable/Lowering/SyntheticBoundNodeFactory.cs

```diff
@@ -1098,6 +1127,15 @@ public BoundExpression Array(TypeSymbol elementType, ImmutableArray<BoundExpress
                 Compilation.CreateArrayTypeSymbol(elementType));
         }
 
+        public BoundExpression Array(TypeSymbol elementType, int elementCount)
```

Comment:
This was used when the initialization of payload arrays was done inline in method prologues. After that initialization moved to a helper method, this method became unreferenced. I will delete it.